components/react/TimeAgo.tsx

```
@@ -0,0 +1,98 @@
+  readonly relativeDate?: TDate;
+}
+
+export const TimeAgo = ({
+  relativeDate,
+  datetime,
+  live = true,
+  mode = TimeAgoMode.Default,
+  tooltip = true,
+  ...timeProps
+}: TimeAgoProps) => {
+  const toString = useCallback(() => {
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)